cmd/stats/commands/stats_cmd.go

```diff
@@ -268,6 +285,24 @@ func printStats(result map[string]stats, header Header) {
 		fmt.Printf("Average received message time for all nodes: %d of total %d is %.2f\n", stats.totalNodesReceivedMsg, header.AgentsCount, stats.sumDurations.Seconds()/float64(stats.totalNodesReceivedMsg))
 		fmt.Printf("Average received message time for validator nodes: %d of total %d is %.2f\n", stats.totalValidatorNodesReceivedMsg, header.ValidatorsCount, stats.sumValidatorsDurations.Seconds()/float64(stats.totalValidatorNodesReceivedMsg))
 	}
+	fmt.Printf("Maximum number validators not received message: %d\n", maxValidatorsNotReceivedMessage)
+	fmt.Printf("Total number of messages that all validator not received: %d (%.2f%%)  messages out of %d messages\n", totalValidatorsNotReceivedAllMsg, float64(100*totalValidatorsNotReceivedAllMsg)/float64(msgNumber), msgNumber)
```

Comment:
maybe: "Total number of messages not received by all validators: %d (%.2f%%)  out of %d messages\n"